reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -48,8 +48,13 @@
 	 * @return the {@link Disposable} instance that let's one cancel this particular task.
 	 * If the {@link Scheduler} has been shut down, throw a {@link RejectedExecutionException}.
 	 */
+	@Deprecated
```

Comment:
please add a `@deprecated` entry to the javadoc explaining the deprecation and alternative